packages/core-utils/src/itinerary.ts

```diff
@@ -788,9 +788,6 @@ export const convertGraphQLResponseToLegacy = (leg: any): any => ({
   agencyUrl: leg.agency?.url,
   alightRule: pickupDropoffTypeToOtp1(leg.dropoffType),
   boardRule: pickupDropoffTypeToOtp1(leg.pickupType),
-  dropOffBookingInfo: {
-    latestBookingTime: leg.dropOffBookingInfo
-  },
```

Comment:
If you're tearing this out of the legacy response, how are you accessing the information instead? Can you add a comment that explains this?